site/soc/software/charts/osh/openstack-compute-kit/neutron.yaml

```diff
@@ -40,34 +34,136 @@ data:
+            kill_metadata35: KillFilter, root, python3.5, -15, -9
+            kill_metadata36: KillFilter, root, python3.6, -15, -9
+            kill_radvd_usr: KillFilter, root, /usr/sbin/radvd, -15, -9, -HUP
+            kill_radvd: KillFilter, root, /sbin/radvd, -15, -9, -HUP
+            
+            # ip_lib
+            ip: IpFilter, ip, root
+            find: RegExpFilter, find, root, find, /sys/class/net, -maxdepth, 1, -type, l, -printf, %.*
+            ip_exec: IpNetnsExecFilter, ip, root
+           
+            # l3_tc_lib
+            l3_tc_show_qdisc: RegExpFilter, tc, root, tc, qdisc, show, dev, .+
+            l3_tc_add_qdisc_ingress: RegExpFilter, tc, root, tc, qdisc, add, dev, .+, ingress
+            l3_tc_add_qdisc_egress: RegExpFilter, tc, root, tc, qdisc, add, dev, .+, root, handle, 1:, htb
+            l3_tc_show_filters: RegExpFilter, tc, root, tc, -p, -s, -d, filter, show, dev, .+, parent, .+, prio, 1
+            l3_tc_delete_filters: RegExpFilter, tc, root, tc, filter, del, dev, .+, parent, .+, prio, 1, handle, .+, u32
+            l3_tc_add_filter_ingress: RegExpFilter, tc, root, tc, filter, add, dev, .+, parent, .+, protocol, ip, prio, 1, u32, match, ip, dst, .+, police, rate, .+, burst, .+, mtu, 64kb, drop, flowid, :1
+            l3_tc_add_filter_egress:  RegExpFilter, tc, root, tc, filter, add, dev, .+, parent, .+, protocol, ip, prio, 1, u32, match, ip, src, .+, police, rate, .+, burst, .+, mtu, 64kb, drop, flowid, :1
+            
+            # For ip monitor
+            kill_ip_monitor: KillFilter, root, ip, -9
+           
+            # ovs_lib (if OVSInterfaceDriver is used)
+            ovs-vsctl: CommandFilter, ovs-vsctl, root
+           
+            # iptables_manager
+            iptables-save: CommandFilter, iptables-save, root
+            iptables-restore: CommandFilter, iptables-restore, root
+            ip6tables-save: CommandFilter, ip6tables-save, root
+            ip6tables-restore: CommandFilter, ip6tables-restore, root
+            
+            # Keepalived
+            keepalived: CommandFilter, keepalived, root
+            kill_keepalived: KillFilter, root, keepalived, -HUP, -15, -9
+            
+            # l3 agent to delete floatingip's conntrack state
+            conntrack: CommandFilter, conntrack, root
+            
+            # keepalived state change monitor
+            keepalived_state_change: CommandFilter, neutron-keepalived-state-change, root
+            # The following filters are used to kill the keepalived state change monitor.
+            # Since the monitor runs as a Python script, the system reports that the
+            # command of the process to be killed is python.
+            # TODO(mlavalle) These kill filters will be updated once we come up with a
+            # mechanism to kill using the name of the script being executed by Python
+            kill_keepalived_monitor_py: KillFilter, root, python, -15
```

Comment:
Is this what we are adding? Can we indicate that so its easier to track and upstream it?

Reply:
Yep, AFAIK only those lines with python3* are new. It's unfortunate we can't really add just a snippet, but the whole config file is a value :-(

And yes, we should add it upstream for sure